sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_ManageModels.md

```diff
@@ -0,0 +1,56 @@
+# Manage models
+
+This sample demonstrates how to manage the models stored in your resource.
+
+To get started you'll need a Cognitive Services resource or a Document Intelligence resource. See [README][README] for prerequisites and instructions.
+
+## Creating a `DocumentIntelligenceAdministrationClient`
+
+To create a new `DocumentIntelligenceAdministrationClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Document Intelligence API key credential by creating an `AzureKeyCredential` object that, if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentIntelligenceAdministrationClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new DocumentIntelligenceAdministrationClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## Manage Models
```

Comment:
nit: Contrary to the other samples, this section is missing a short overview of the code snippet.

Reply:
Totally missed it, somehow. Will add it before merging the PR.